crypto/test/sysgenid_test_util.cc

```diff
@@ -0,0 +1,97 @@
+// -----------------------------------------------------------------------------
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+// -----------------------------------------------------------------------------
+
+#include "sysgenid_test_util.h"
+
+#if defined(OPENSSL_LINUX)
+
+#include <stdio.h>
+
+#include "../fipsmodule/rand/snapsafe_detect.h"
+
+#include <fcntl.h>
+#include <sys/ioctl.h>
+#include <sys/stat.h>
+#include <string.h>
+#include <unistd.h>
+
+// These values can be found here: https://lkml.org/lkml/2021/3/8/677.
+#define SYSGENID_IOCTL                  0xE4
+#define SYSGENID_TRIGGER_GEN_UPDATE     _IO(SYSGENID_IOCTL, 3)
+
+static int system_supports_snapsafe = SNAPSAFE_NOT_SUPPORTED;
+
+// |set_sysgenid_file_value| interfaces with the SysGenID device. If this is not
```

Comment:
When searching the PR changes for `set_sysgenid_file_value`, I only find it mentioned here. What is it referring to?

Reply:
Something I forgot to delete... (: